Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -390,44 +390,92 @@ public extension Tensor {
         return Raw.gatherV2(params: self, indices: indices, axis: Tensor<Int32>(Int32(axis)))
     }
 
-    /// Returns slices of this tensor at `indices`, while ignoring the first `batchDims` dimensions
-    /// that correspond to batch dimensions. The gather is performed along the first non-batch
-    /// dimension.
+    /// Returns slices of this tensor at `indices` along the `axis` dimension, while ignoring the 
+    /// first `batchDims` dimensions that correspond to batch dimensions. The gather is performed 
+    /// along the first non-batch dimension.
     ///
     /// Performs similar functionality to `gathering`, except that the resulting tensor shape is 
-    /// now:
-    /// ```
-    /// self.shape[..<batchDims] + 
-    ///   indices.shape[batchDims...] + 
-    ///   self.shape[(batchDims + indices.rank + 1)...]
-    /// ```
+    /// now `self.shape[..<axis] + indices.shape[batchDims...] + self.shape[(axis + 1)...]`.
     ///
     /// - Parameters:
     ///   - indices: Contains the indices to gather.
+    ///   - axis: Dimension along which to gather. Negative values wrap around.
     ///   - batchDims: Number of leading batch dimensions to ignore.
     ///
+    /// - Precondition: `axis` must be in the range `[-rank, rank)`, while also being greater than
```

Comment:
Use Swift syntax for ranges.
```suggestion
    /// - Precondition: `axis` must be in the range `-rank..<rank`, while also being greater than
```